changes.d/6623.fix.md

```diff
@@ -0,0 +1,4 @@
+Auto restart: The "force condemn" option (that tells workflows running on a
+server to shutdown as opposed to migrate) hasn't worked with the host-selection
+mechanism since Cylc 8.0.0. This has now been fixed and the "force condemn"
+option has been restored in the documentation.
```

Comment:
```suggestion
Restored the option to shut down rather than auto-migrate workflows on a condemned
host. This option hasn't worked with the host-selection mechanism since Cylc 8.0.0. 
```

Reply:
This isn't actually correct.